roles/etcd/templates/etcd-docker.service.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=etcd docker wrapper
-Wants=docker.socket
-After=docker.service
+Wants=docker.service docker.socket
```

Comment:
I'm entirely positive. Check out commit aa0d7ea5d0e9f23598d6ddf238465e3fe4c7b898 and look at /var/log/syslog after each systemd reload and how many times docker daemon restarts. Compare it with master